dpcpp/test/base/executor.dp.cpp

```diff
@@ -60,31 +60,42 @@ class DpcppExecutor : public ::testing::Test {
 
     void SetUp()
     {
-        ASSERT_GT(gko::DpcppExecutor::get_num_devices("cpu"), 0);
-        dpcpp = gko::DpcppExecutor::create(0, omp, "cpu");
         if (gko::DpcppExecutor::get_num_devices("gpu") > 0) {
```

Comment:
Is `get_num_devices()` affected by DEVICE_FILTER?

Reply:
Everything is affected by `SYCL_DEVICE_FILTER` or `SYCL_DEVICE_TYPE`. If you set `CPU` as the filter/type, there will always be 0 GPU found, which is why I need to write this test like this.

Reply:
I see. Thanks